Resources/Maps/Shuttles/loader.yml

```diff
@@ -12,28 +13,27 @@ entities:
   entities:
   - uid: 1
     components:
-    - name: NT14 Loader PR-712
+    - name: grid
```

Comment:
Not sure how this happens, but this is suspicious. How did you save the grid? Which command did you use?

Reply:
The file was a map to start with, this is why its like this
This is a rebuild of the ship into grid from 0

Reply:
Yeeep, that would make sense... It's overridden by the shipyard system anyways on init, so this is fine. Resolved.